pnpm-workspace.yaml

```diff
@@ -1,4 +1,3 @@
 packages:
-  - "packages/hub"
-  - "packages/inference"
-  - "packages/doc-internal"
+  - "packages/*"
+  - "!packages/doc-internal"
```

Comment:
It's still part of the workspace and we still want linter to run on it in the lint CI. 

You should probably exclude it inside `.changeset/config.json`, in the `ignore` setting instead.